cmd/influxd/run/server.go

```diff
@@ -645,7 +645,10 @@ func (s *Server) stopProfile() error {
 type monitorPointsWriter coordinator.PointsWriter
 
 func (pw *monitorPointsWriter) WritePoints(database, retentionPolicy string, points models.Points) error {
-	return (*coordinator.PointsWriter)(pw).WritePointsPrivileged(database, retentionPolicy, models.ConsistencyLevelAny, points)
+	writeCtx := tsdb.WriteContext{
```

Comment:
Is it okay to have something named` WriteContext` which doesn't have a context.Context inside of it?

Reply:
`context.Context` does two things:

* cancellation
* request-scoped values

I really dislike using the request-scoped-values and I think they are mostly an antipattern. Here we need request scoped values but not cancellation. See e.g. https://lestrrat.medium.com/alternative-to-using-context-value-f2efe6bd2788 - I'm not alone.

As written this is a bit ugly though, I should probably either move `writeCtx` to be the first argument, or rename it to something like `WriteDetails`.

Reply:
Note we could put a `context.Context` inside `tsdb.WriteContext` as a member, if we want cancellation later.

Reply:
I'm also not a fan of passing request-scoped-values through `context.Context` even though that's partially what it was designed for and is common practice in Go. I find that code harder to follow.

Naming is tough. Anything with `Context` in the name in Go will initially make most people think of `context.Context` but I don't think that should discourage using `Context` in a type name where it best describes the type. `WriteContext` seems okay to me here.